Assignment4/Josesilva99-z170360/Task06.ipynb

```diff
@@ -319,6 +319,7 @@
         "#Comprobamos:\n",
         "for s, p, o in g.triples((ns.UPM, RDF.type, ns.University)):\n",
         "  print(s,p,o)\n",
+        "VCARD = Namespace(\"http://www.w3.org/2006/vcard/ns#\")\n",
```

Comment:
well, this is not the way you have to resolve this issue. VCARD is a common and shared vocabulary to define relations and information about people, it doesn't contain any property called WORK, so use another namespace please, ns for example... I suppose that you understand the reason with this explanation